### PR TITLE
Mobile nav toggle

### PR DESCRIPTION
**Pull request by sediqullahbadakhsh**
With respect to Figma Design added the following functionalities to the Mobile Navbar:
1- toggle functionality.
2- clicking on link the navbar closes (disappears)
3- clicking on a link, moves you to the particulare section
4- clicking on humberger changes to X and viseversa
5- while openning Navbar added transations
6- set mobile-navbar to position fixed
7- while opening navbar, the logo gets hidden.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+  burger.addEventListener('click', () => {
+    nav.classList.toggle('nav-active');
+    burger.classList.toggle('toggle');
+  });
+  document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click', () => {
+    nav.classList.remove('nav-active');
+    burger.classList.remove('toggle');
+  }));
+};
```
Comment (mateo951) [inline app.js:13]: The menu is working perfectly 🔥 Good job guys 👌

File: style.css [hunk 2]
```diff
@@ -48,13 +46,66 @@ span {
...
+  align-items: center;
+  z-index: 1000;
+  right: 0;
+  height: 100vh;
+  top: 48px;
+  background-color: #3e3c3c;
+  width: 100%;
+  padding: 0 24px;
+  transform: translateX(100%);
+  transition: transform 0.5s ease-in;
```
Comment (mateo951) [inline style.css:95]: Transitions are looking good 🙌

**Review by mateo951 — APPROVED** [review]
### Hi Team, 

👌 **Your project highlights** 😎 
- General Requirements 🔥
- Using JS best practices 👌
- Menu design 😎

You have done very good work implementing all the project requirements 🤓
Your project is complete!  There is nothing else to say other than... it's time to merge it 😎
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.js (hunk 1 above)
Comment (sediqullahbadakhsh) [reply app.js:13]: Thank you so much

**Pull request merged**